README.md

```diff
@@ -38,6 +34,17 @@ in the same bitness as your Node.js installation.
    * Either install Microsoft's [windows-build-tools](https://github.com/felixrieseberg/windows-build-tools) using `npm install --global --production windows-build-tools`
    * Or manually install [Python 2.7](http://www.python.org/download/) and [Visual Studio](https://www.visualstudio.com/downloads/) (or modify an existing installation) and select Common Tools for Visual C++ during setup.
 
+## Enabling the SDK
+To import the Yoti SDK inside your project, you can use your favourite dependency management system.
+If you are using NPM, you need to add the following dependency:
```

Comment:
I would just say:

`npm install -S yoti-node-sdk`

Rather than showing to alter the `package.json` directly.